package.json

```diff
@@ -4,7 +4,7 @@
   "description": "transparently require() jsx from node",
   "main": "index.js",
   "dependencies": {
-    "react-tools": "^0.13.2"
+    "jstransform": ">=11.0.0"
```

Comment:
I'm always hesitant to use `>=` in versions. This led to a big mess with `reactify` when we changed the JSX transform. We could change a default, ship a new major version, and then one person's build is vastly different from another's.